topic_template_testC.txt

```diff
@@ -1 +0,0 @@
-{{- .Path -}}_{{- .Identifier -}}
```

Comment:
Why was this file deleted?

Reply:
My bad, just restored it. Looks like I deleted that file (`topic_template_testC.txt`) instead of `vendor.patch`.